LOAN/LoanContracts/LoanApplications.rdf

```diff
@@ -112,14 +152,140 @@
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-oc;hasInput"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-crt;CreditReport"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-oc;hasInput"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-app;PreApprovalContract"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-oc;hasInput"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-app;PreApprovalRequest"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-oc;hasInput"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-app;PublicRecord"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-oc;hasOutput"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-app;UnderwritingDecision"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;appliesTo"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;Loan"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-app;usesFactor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-app;AllBorrowersMonthlyIncome"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-app;usesFactor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-app;BorrowerMonthlyIncome"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-app;usesFactor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-app;TotalDebtExpenseRatio"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-app;usesFactor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;CombinedLoanToValueRatio"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-cr;isClassifiedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-app;UnderwritingAutomation"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>credit risk assessment</rdfs:label>
+		<skos:definition>risk assessment that focuses on determining the likelihood of a potential borrower repaying a loan</skos:definition>
+		<fibo-fnd-utl-av:usageNote>If the risk assessment is based on one of the automated underwriting sytems, then the underwriting automation category is &apos;automated&apos;. This dependency could be automated (as it were).</fibo-fnd-utl-av:usageNote>
```

Comment:
This again came from the work that MU did - I moved it out of LoanCore so that we could do some work with commercial lenders in an FCT effort - not sure it is applicable to all kinds of loans, and it seems to be very US / HMDA and consumer focused.  Let's leave this for work by SMEs.